aldryn_django/i18n_urls.py

```diff
@@ -14,3 +17,7 @@
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
 ]
+
+# serve static files differently on local development
+if 'runserver' in sys.argv:
```

Comment:
Maybe also add a check for `and not settings.MEDIA_URL_IS_ON_OTHER_DOMAIN` https://github.com/divio/aldryn-django/blob/support/2.1.x/aldryn_config.py#L380

Reply:
adapted :)